package.json

```diff
@@ -1,10 +1,18 @@
 {
```

Comment:
- [ ] Here, you are missing the `react-dom` and `babel-loader` dependencies for the front-end(React) which is required.